src/main/java/racingcar/domain/Cars.java

```diff
@@ -23,29 +30,38 @@ public void addCar(Car car) {
     }
 
     public void moveAllCar() {
+        Strategy numberStrategy = new NumberStrategy();
         for (Car car : cars) {
-            car.stopOrMove(getRandomValue());
+            car.stopOrMove(numberStrategy);
         }
     }
 
-    private int getRandomValue() {
-        Random random = new Random();
-        return random.nextInt(10);
-    }
-
-    public List<Integer> getAllMoveCount() {
-        List<Integer> moveCounts = new ArrayList<>();
+    public final List<Car> getAllCar() {
+        List<Car> copyCars = new ArrayList<>();
         for (Car car : cars) {
-            moveCounts.add(car.getMoveCount());
+            copyCars.add(car.clone());
         }
-        return moveCounts;
-    }
-
-    public List<Car> getAllCar() {
-        return cars;
+        return copyCars;
     }
 
     public int getNumberOfCar() {
         return cars.size();
     }
+
+    public List<String> findWinners() {
+        List<String> winners = new ArrayList<>();
+        cars.sort(Comparator.comparingInt(Car::getMoveCount).reversed());
+        int winnerMoveCount = cars.get(0).getMoveCount();
```

Comment:
```suggestion
        cars.sort(Car::compareTo);
        int winnerMoveCount = cars.get(cars.size() - 1).getMoveCount();
```

Car 자체를 비교할 수 있도록 Car에 Comparable을 구현할 수 있어보입니다.